src/components/list-item/ListItem.tsx

```diff
@@ -118,26 +50,28 @@ export const ListItem = forwardRef<HTMLDivElement, Props>(
     },
     ref,
   ) => (
-    <AminoListItem
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+    <div
       ref={ref}
-      className={className}
-      disabled={disabled}
+      className={clsx(
```

Comment:
I see quite a few places having this comment eslint disable `jsx-a11y/no-static-element-interactions`. To not to have this comment, as the rule suggest [here](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-static-element-interactions.md#succeed), you may want to add `role="button"` to any static element that has `onClick` event on it.
Ex: 
```
<div
    role="button"
    ref={ref}
    ...
```

Reply:
When doing this, I still get "elements with click handlers must have at least one keyboard listener. Should I add `onKeyDown={() => {}}` or listen for the enter key to 'click' or something else?